test/LEGO.AsyncAPI.Tests/Models/AsyncApiSchema_Should.cs

```diff
@@ -320,7 +320,7 @@ public class AsyncApiSchema_Should
       description: test";
 
         private string InlinedReferences =>
-            @"asyncapi: '2.6.0'
+            @"asyncapi: 2.6.0
```

Comment:
This was part of the reason the tests were broken. Some version like objects were quoted and other were not. Based on the yaml spec as well as the examples from AsyncApi I went with their lead, they add no quotes.

Reply:
i think originally we erred on the side of quoting, but I think this change is good.
Also really liking the additions of the missing cases

Reply:
Thinking a bit harder on this.
This might end up causing issues.
as number styles differ from `,` to `.`.
having this as a number specifically, will mean that it might end up not parsing on certain machines.
depending on culture.
enforcing is as a non-ambiguous string, removes this concern completely.